src/org/exist/xquery/value/ValueSequence.java

```diff
@@ -29,6 +29,7 @@
 import org.exist.numbering.NodeId;
 import org.exist.util.FastQSort;
 import org.exist.xquery.*;
+import org.exist.xquery.NodeTest; // Needed by Eclipse to disambiguate
```

Comment:
Any idea what the conflict is? maybe a unit directory is not marked as a test directory? For me, this is kinda suspicious.

Reply:
Eclipse error : 
The type NodeTest is ambiguous	/eXist/src/org/exist/xquery/value	ValueSequence.java	line 564	Java Problem

Eclipse suggest to choose between many :
import com.sun.org.apache.xalan.internal.xsltc.compiler.NodeTest;
import com.sun.org.apache.xpath.internal.patterns.NodeTest;
import net.sf.saxon.pattern.NodeTest;
import org.apache.xalan.xsltc.compiler.NodeTest;
import org.apache.xpath.patterns.NodeTest;
...
import org.exist.dom.persistent.NodeTest;
import org.exist.fluent.NodeTest;
import org.exist.xquery.NodeTest;

Only the last one fits.

Reply:
This sounds like an iffy problem in the Eclipse Java compiler. If it helps Eclipse users, then adding the import seems fine to me.